openapi/src/main/java/io/micronaut/openapi/visitor/AbstractOpenApiVisitor.java

```diff
@@ -2015,6 +2015,15 @@ protected Schema readSchema(AnnotationValue<io.swagger.v3.oas.annotations.media.
     private List<Object> getEnumValues(EnumElement type, String schemaType, String schemaFormat, VisitorContext context) {
         List<Object> enumValues = new ArrayList<>();
         for (EnumConstantElement element : type.elements()) {
+
+            AnnotationValue<io.swagger.v3.oas.annotations.media.Schema> schemaAnn = element.getAnnotation(io.swagger.v3.oas.annotations.media.Schema.class);
+            boolean isHidden = schemaAnn != null && schemaAnn.get("hidden", Boolean.class).orElse(false);
```

Comment:
can be written as `element.booleanValue(io.swagger.v3.oas.annotations.media.Schema.class, "hidden").orElse(false)`

Reply:
@graemerocher fixed